snapcraft/internal/repo/_deb.py

```diff
@@ -54,6 +55,8 @@
 '''
 _GEOIP_SERVER = "http://geoip.ubuntu.com/lookup"
 _library_list = dict()  # type: Dict[str, Set[str]]
+_HASHSUM_MISMATCH_PATTERN = re.compile(
+    r'(E:Failed to fetch copy.+Hash Sum mismatch)+')
```

Comment:
You can probably omit the "copy" here, this refers to the internal helper that generated the error and it might change in the future.

Reply:
Done. Thanks for the review!